lib/pg_online_schema_change/orchestrate.rb

```diff
@@ -111,13 +113,13 @@ def setup_trigger!
           $$
           BEGIN
             IF ( TG_OP = 'INSERT') THEN
-              INSERT INTO \"#{audit_table}\" select 'INSERT', now(), NEW.* ;
+              INSERT INTO \"#{audit_table}\" select nextval(\'#{audit_table_pk_sequence}\'), 'INSERT', now(), NEW.* ;
```

Comment:
Non blocking since this is how it was before this PR, but I wonder if we should use `clock_timestamp ( )` instead of `now()`: https://www.postgresql.org/docs/14/functions-datetime.html

Reply:
yeah I think we def can, better to have more accuracy here. There is also no current use for these right now. I found them useful when building and debugging, so still keeping it in case users find it useful when debugging broken operations.

Reply:
done